pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -55,47 +55,20 @@ stakeholders.
 
 Available integrations are:
 
-- <a href="#segment-protocols">Segment Protocols</a>
-- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
-- <a href="#rudderstack">RudderStack</a>
-- <a href="#amplitude-govern">Amplitude Govern</a>
-- <a href="#snowplow">Snowplow Data Structures</a>
-- <a href="#mparticle">mParticle Data Master</a>
-- <a href="#webhook">Webhook</a>
```

Comment:
I really like this change. The only downside I can think of is that these old links will now be invalid. We'll have to remember to update our links from the web app tooltips and point them to the new locations.

Reply:
Riiiighht. So we may want to wait with merging this one until we have updated the links in the app, right? I only have read access to the monorepo so I need some help there.

Reply:
I don't think we need to block the release of this, just make sure we don't let too long time lapse between fixing the links.